W-15135730 GA branch for late Feb release jh (#23)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/getting-started-with-api-experience-hub.adoc

= Getting Started with API Experience Hub 
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: prerequisites.adoc

Verify that you have the following required accounts and access:

* Anypoint Platform organization in production
* Salesforce organization
* Organization Administrator of the main organization

> * You must have the System Administrator profile or a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].

Reviewer: ```suggestion
* You must have the System Administrator profile or be a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].
```

Reviewer: Not sure if that's correct.

Reviewer: Does a profile get assigned? If so, then maybe something like:
* You must be assigned either the System Administrator profile ...

Reviewer: This is hard to parse. I wonder if a sublist is in order.

You must either be:
* Assigned the System Administrator profile
* A Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process.

* Enterprise Edition
* Performance Editions of Salesforce

> If you do not have the required accounts and access, issues can occur during the Salesforce linking process when creating your API portal.

Reviewer: ```suggestion
If you don't have the required accounts and access, issues might occur during the Salesforce linking process when creating your API portal.
```
